install.php

```diff
@@ -2,7 +2,40 @@
 
```

Comment:
Hier ggf deine tolle PR beschreibung als kommentar in den source damit man den Hintergrund versteht

Reply:
Erledigt, danke!